jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/TaskQueryServiceBaseTest.java

```diff
@@ -270,6 +278,234 @@ public void testGetTasksAssignedAsPotentialOwnerWithUserGroupsLangStatus() {
     }
     
     
+    @Test
+    public void testGetTasksAssignedAsPotentialOwnerWithNoUserGroupsLangStatus() {
+        startBaseTest();
+        final List<String> groups = new ArrayList<String>();
```

Comment:
Explicit type argument not really needed
```suggestion
        final List<String> groups = new ArrayList<>();
```

Reply:
removed